tests/orquestra/vqa/algorithms/vqe_test.py

```diff
@@ -172,9 +178,13 @@ def test_find_optimal_params_doesnt_fail(
     def test_get_cost_function(self, vqe_object, simulator):
         cost_function = vqe_object.get_cost_function(simulator)
         assert np.isclose(
-            cost_function(np.zeros(vqe_object.ansatz.number_of_params)), 0
+            cost_function(np.zeros(vqe_object.ansatz.number_of_params)), 1
         )
 
+    def test_get_cost_function_with_shots(self, vqe_object_with_shots, simulator):
+        vqe_object_with_shots.get_cost_function(simulator)
+        # TODO: test behavior of get_cost_function
```

Comment:
TODO needs to be fixed.

Reply:
I was imagining to address this in a future story. It seems like figuring out a way to properly test VQAs with sampling is kind of a non-trivial problem that probably shouldn't block this bugfix. Wdyt?

Reply:
I think it might be better to delete TODO and create a story.
It's better not to proliferate TODOs in the code itself, as per the conversations we had with the SDK team.

Reply:
Done!